common/beerocks/bwl/shared/nl80211_client_impl.cpp

```diff
@@ -335,6 +335,10 @@ bool nl80211_client_impl::get_radio_info(const std::string &interface_name, radi
                         channel.is_dfs = true;
                     }
 
+                    if (tb_freq[NL80211_FREQUENCY_ATTR_NO_IR]) {
+                        channel.no_ir = true;
+                    }
```

Comment:
In the future we would want to check if radar detection is supported and if not these NO_IR (DFS) channels should not be included in the AP capabilities.

This brings me to the next point, 
I suggest that we  add `is_radar_detection_supported`  flag to the AP class , IMO should be read driver but maybe for now it can be read from prplmesh_db (need to add it there also) @arnout  what do you think
Then add this  NO_IR channels check :

```
                    // Ignore DFS channels in case AP doesn't support radar detection.
                    if (channel.no_ir && is_radar_detection_supported) {
                        continue;
                    }
```

Reply:
I believe a better check is to check the `is_dfs` tag as well as the `no_ir` flag